scripts/measures/enrich-measures-data.js

```diff
@@ -0,0 +1,74 @@
+const fs = require('fs');
+const path = require('path');
+const parse = require('csv-parse/lib/sync');
+
+const aciRelations = require('../../util/measures/aci-measure-relations.json');
+const cpcPlusGroups = require('../../util/measures/cpc+-measure-groups.json');
+
+const qpp = fs.readFileSync(path.join(__dirname, '../../staging/measures-data.json'), 'utf8');
```

Comment:
What is the staging data file from?

Reply:
The staging data file is the version of `measures-data.json` currently on the `master` branch.  https://github.com/CMSgov/qpp-measures-data/blob/65864866f828c349ef41e55f9b42e69d457492f9/measures/measures-data.json#L1

We've opted to place this original copy of `measures-data.json` in `staging` and then apply the enriching functions to the original copy to generate the version in `measures/measures-data.json`.  Let me know if you'd like this to be changed @marimiyachi.

cc: @bijujoseph

Reply:
Gotcha. Thanks!